CoinChangerUnitTests/CoinChanger.cs

```diff
@@ -0,0 +1,59 @@
+using System.Collections;
+using System.Collections.Generic;
+using System;
+using System.Linq;
+
+namespace coinChanger
```

Comment:
For name spaces you will want to use Pascal Case. So `coinChanger` => `CoinChanger`

Reply:
Ah good to know!